src/dune_engine/print_diff.ml

```diff
@@ -59,17 +59,17 @@ end = struct
     | { commands = []; error } -> raise (User_error.E error)
     | { commands = { dir; metadata; prog; args } :: commands; error } ->
       let* () =
-        Process.run ~display:!Clflags.display ~dir ~env:Env.initial Strict prog
-          args ~metadata
+        Process.run ~display:Quiet ~dir ~env:Env.initial Strict prog args
+          ~metadata
```

Comment:
This executes a list of programs together with some meta data for when we run our diffing program. We do not need to report the actual process of the diffing program to the user.